File: app/src/main/java/com/BlizzardArmory/warcraft/Talents/SpecializationData/Media.java

package com.BlizzardArmory.warcraft.Talents.SpecializationData;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Media {

    @SerializedName("key")
    @Expose
    private Key key;
    @SerializedName("id")
    @Expose
    private int id;

    /**
     * No args constructor for use in serialization
     */
    public Media() {
    }

    /**
     * @param id
     * @param key
     */
    public Media(Key key, int id) {
        super();
        this.key = key;
        this.id = id;
    }

    public Key getKey() {
        return key;
    }

    public void setKey(Key key) {
        this.key = key;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

}

File: app/src/main/java/com/BlizzardArmory/model/warcraft/talents/specializationdata/Media.kt
package com.BlizzardArmory.model.warcraft.talents.specializationdata

import com.google.gson.annotations.SerializedName


/**
 * The type Media.
 */
data class Media(

        @SerializedName("key")
        var key: Key,

        @SerializedName("id")
        var id: Long
)